### ISSUE TITLE
License Clarification

### ISSUE THREAD

**Issue by JoyceBabu** [browserless/browserless#3850]
While exploring version 2 of the software, I noticed a shift in the project license from GPLv3 to SSPL-1.0. However, the issue highlighted in [this GitHub issue](https://github.com/browserless/browserless/issues/1791) persists.

The LICENSE file states:

> If you want to use Browserless to build commercial sites, applications, or in a continuous-integration system that's closed-source then you'll need to purchase a commercial license.
> If you are creating an open source application under a license compatible with the Server Side License 1.0, you may use Browserless under those terms.

However, the SSPL-1.0 license does not mandate that all projects utilizing the software must be open source. It specifically targets scenarios where the project is offered as a SaaS. This approach was notably adopted by MongoDB to address services like AWS, which offered MongoDB as a SaaS, rather than commercial projects utilizing MongoDB internally. This distinction is why MongoDB designed SSPL based on the GPL rather than the AGPL.

Therefore, it appears that under SSPL, one is permitted to use the unmodified docker container and integrate the self-hosted service with their own software or website. However, creating a direct competitor to Browserless.io without open-sourcing the competing service's code is prohibited.

If Browserless aims to prevent commercial exploitation of its software without financial compensation, that's entirely justifiable. However, employing a license that allows for commercial use, while simultaneously issuing statements that suggest otherwise, generates confusion. A clarification on this matter would be greatly appreciated.

**Comment by igorer88** [comment]
I agree totally, a clarification would be great for startups that just want to use the product for some of its features.

**Comment by moltar** [comment]
Agreed. This is very muddy. I just wanted to use it for e2e tests internally. Verbiage states that integration tests require a purchase of a license. This doesn't make sense. Mongo use case in an integration test would have been free.

**Comment by vvolodin** [comment]
Any update on this?

**Comment by riddla** [comment]
How was this completed?

**Comment by SpencerMelo** [comment]
Any actual answers on this?
